.packit.yaml

```diff
@@ -0,0 +1,40 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-smart_proxy_pulp.spec
+
+# add or remove files that should be synced
+files_to_sync:
+  - rubygem-smart_proxy_pulp.spec
+  - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: smart_proxy_pulp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-smart_proxy_pulp
+
+upstream_tag_template: '{version}'
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec -O rubygem-smart_proxy_pulp.spec"
+  get-current-version:
+    - ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version'
+  create-archive:
+    - gem build smart_proxy_pulp.gemspec
+    - bash -c "ls -1t ./smart_proxy_pulp-*.gem | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      rhel-9:
+        additional_modules: "foreman-devel:el9"
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el9/x86_64/
+          - https://yum.theforeman.org/plugins/nightly/el9/x86_64/
+    module_hotfixes: true
```

Comment:
Do we need this without el8?

Reply:
Need? No.
Will it be useful in the future if we ever decide to flip Ruby to anything other than 3.0 and not have to touch every single packit yaml? Yes :)